tests/suites/cassandra_tls/tls_test.go

```diff
@@ -0,0 +1,214 @@
+				"cassandra",
+			)
+			Expect(output).To(ContainSubstring("Starting Encrypted Messaging Service on SSL port"))
+			// Expect(output).To(ContainSubstring("Enabling encrypted CQL connections between client and server"))
+		})
+		It("Test data read & write using CQLSH", func() {
+			output, err := cassandra.CQLSH(
+				TestNamespace,
+				TestInstance,
+				testCQLScript,
+			)
+			Expect(err).To(BeNil())
+			Expect(output.String()).To(ContainSubstring(testCQLScriptOutput))
+		})
+		It("Uninstalls the operator", func() {
+			err := kudo.UninstallOperator(OperatorName, TestNamespace, TestInstance)
+			Expect(err).To(BeNil())
+			// TODO(mpereira) Assert that it isn't running.
+		})
+	})
+
+	Context("Installs the operator with client-to-node encryption enabled", func() {
+		It("Installs the operator from a directory", func() {
+			err := kudo.InstallOperator(
+				OperatorDirectory, TestNamespace, TestInstance, []string{
+					"NODE_CPU_MC=200",
+					"NODE_MEM_MIB=800",
+					"PROMETHEUS_EXPORTER_CPU_MC=100",
+					"PROMETHEUS_EXPORTER_MEM_MIB=200",
+					"TLS_SECRET_NAME=cassandra-tls",
+					"TRANSPORT_ENCRYPTION_CLIENT_ENABLED=true",
+				},
+			)
+			if err != nil {
+				Fail(
+					"Failing the full suite: failed to install operator instance that the " +
+						"following tests depend on",
+				)
+			}
+			Expect(err).To(BeNil())
+			assertNumberOfCassandraNodes(NodeCount)
+		})
+		It("Check container logs", func() {
+			output, _ := k8s.FetchLogsOfPod(
+				TestNamespace,
+				fmt.Sprintf("%s-%s-%d", TestInstance, "node", 0),
+				"cassandra",
+			)
+			Expect(output).To(ContainSubstring("Enabling encrypted CQL connections between client and server"))
+		})
+		It("Test data read & write using CQLSH", func() {
+			output, err := cassandra.CQLSH(
+				TestNamespace,
+				TestInstance,
+				testCQLScript,
+			)
+			Expect(err).To(BeNil())
+			Expect(output.String()).To(ContainSubstring(testCQLScriptOutput))
+		})
+		It("Uninstalls the operator", func() {
+			err := kudo.UninstallOperator(OperatorName, TestNamespace, TestInstance)
+			Expect(err).To(BeNil())
+			// TODO(mpereira) Assert that it isn't running.
+		})
+	})
+
+	Context("Installs the operator with node-to-node and client-to-node encryption enabled", func() {
+		It("Installs the operator from a directory", func() {
+			err := kudo.InstallOperator(
+				OperatorDirectory, TestNamespace, TestInstance, []string{
+					"NODE_CPU_MC=200",
+					"NODE_MEM_MIB=800",
+					"PROMETHEUS_EXPORTER_CPU_MC=100",
+					"PROMETHEUS_EXPORTER_MEM_MIB=200",
+					"TLS_SECRET_NAME=cassandra-tls",
+					"TRANSPORT_ENCRYPTION_ENABLED=true",
+					"TRANSPORT_ENCRYPTION_CLIENT_ENABLED=true",
+				},
+			)
+			if err != nil {
+				Fail(
+					"Failing the full suite: failed to install operator instance that the " +
+						"following tests depend on",
+				)
+			}
+			Expect(err).To(BeNil())
+			assertNumberOfCassandraNodes(NodeCount)
+		})
+		It("Check container logs", func() {
```

Comment:
Try to follow the active voice for the test names, so that it reads like:
```
It "checks for the container logs"
```
like the above
```
It "installs the operator from a directory"
```
On the other tests as well.